reference/CloudPay-API-Specification.yaml

```diff
@@ -2251,10 +2251,15 @@ components:
             type: string
             format: date-time
             readOnly: true
+          Type:
+            type: string
+            enum: ['recurring', 'non-recurring']
```

Comment:
my understanding of the ticket was that we need to show the type of subscription right? so maybe explicit types instead of flag?

Reply:
we can also do integers for any enums if you feel like we should decrease the size of the payload